cpu/esp8266/Makefile.include

```diff
@@ -0,0 +1,165 @@
+# check some environment variables first
+ifndef NEWLIB_DIR
```

Comment:
please change to, e.g., ESP_NEWLIB_DIR

Reply:
@kaspar030 of course, good point

Reply:
@kaspar030 We would also have to change the riotdocker file accordingly.

Reply:
@kaspar030 No, the ```riotdocker``` file does not need to be changed because the ```SDK_DIR``` and ```NEWLIB_DIR``` variables are not set in the ```riotdocker``` environment. The default settings of ```cpu/esp8266/Makefile.include``` are used instead.

At this point, I wonder if it is really necessary to change the variable names in ```ESP8266_NEWLIB_DIR``` and ```ESP8266_SDK_DIR```. If the user does not explicitly set ```NEWLIB_DIR``` and ```SDK_DIR```, they are set to the default values only inside the make environment of the current board, which in this case is always the esp8266 tool environment. If the user explicitly sets them, he probably wants to compile only esp8266 boards.

Reply:
I'd insist on renaming, since at least on OS X the toolchain install process is quite specific, which means I cannot reuse the binaries on your repo directly, but only newlib. I put my toolchain in a very different directory so I defined the paths by myself.

This might be the case for other developers.

Reply:
Done